sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -56,103 +73,85 @@
   </v-navigation-drawer>
 </template>
 <script lang="ts">
-import { useRouter } from "vue-router";
+import { ref } from "vue";
 import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
 import { MenuModel, Role } from "@/types";
 import CheckPermissionRole from "@/components/generic/CheckPermissionRole.vue";
 
 export default {
   components: { CheckPermissionRole },
   setup() {
-    const router = useRouter();
+    const drawer = ref("drawer");
     const topItems = [
       {
         label: "Home",
         icon: "mdi-home-outline",
-        command: () => {
-          router.push({
-            name: AESTRoutesConst.AEST_DASHBOARD,
-          });
-        },
+        command: () => ({
+          name: AESTRoutesConst.AEST_DASHBOARD,
+        }),
```

Comment:
The existing code doesn't hurt. But title and icon can be set in the JSON itself like this. So that we don't have to re-assign them as attribute value in the component.

![image](https://user-images.githubusercontent.com/54600590/188209280-90c79c16-5a9e-4577-bfdd-a50d80428813.png)

Reply:
@andrewsignori-aot @ann-aot @guru-aot what is your opinion?

Reply:
Sorry, I am not following it, can you please give an example of how it would improve the current code?

Reply:
Got it now after our chat. Looks better to me also 👍

Reply:
As discussed 😊

Reply:
For now not implementing as the :to property cannot be assigned to the items. Navigation is not supported.

Reply:
We can give it a try later in upcoming PR/UI stories.